parm/config/gefs/config.resources

```diff
@@ -188,7 +188,7 @@ case ${step} in
 
     case "${CASE}" in
       "C48" | "C96" | "C192")
-        declare -x "walltime"="03:00:00"
+        declare -x "walltime"="08:00:00"
```

Comment:
Is the longer walltime due to forecast length? Would GEFS also need 8hrs?

Reply:
GEFS does not run these resolutions. Forecast time is for long SFS forecast, but I rolled it back to 03 if needed for CI testing. An easy method to change this would be nice.

Reply:
Gotcha, thanks for explaining!